ctadata/api.py

```diff
@@ -113,6 +141,50 @@ def fetch_and_save_file(self, path, save_to_fn=None):
 
         return total_wrote
 
+    def upload_certificate(self, certificate_file):
+        try:
+            certificate = open(certificate_file, 'r').read()
+        except FileNotFoundError:
+            raise FileNotFoundError('Certificat file not found')
+
+        url = self.construct_endpoint_url('upload-cert', None)
+        r = requests.post(url,
+                          json={'certificate': certificate},
+                          headers={
+                              "HTTP_USER_AGENT": "CTADATA-"+__version__,
+                              "AUTHORIZATION": 'Bearer '+(self.token or ''),
+                          })
+        if r.status_code == 200:
+            return r.json()
+        else:
+            raise CertificateError(r.text)
+
+    def upload_admin_certificate(self, certificate_file, cabundle_file=None):
```

Comment:
I prefer this, since "file" could rather an mean open file.

```suggestion
    def upload_admin_certificate(self, certificate_file_path, cabundle_file_path=None):
```